torch/_meta_registrations.py

```diff
@@ -720,6 +720,12 @@ def sym_constrain_range_for_size(size, min=None, max=None):
 
     if isinstance(size, (SymFloat, SymBool)):
         raise ValueError("Constraining SymFloat or Symbool is nyi")
+    if type(size) is int:
+        if min is not None:
+            torch._check(size >= min)
+        if max is not None:
+            torch._check(size <= max)
```

Comment:
just curious, why did we not need these extra torch._checks before?

Reply:
So previously it was an error to pass a concrete int into this function (constrain range for size would fail). I am making it not fail by just immediately doing the test here.